packages/notification-center/src/store/styles/styles-provider.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent, createContext, useMemo, useContext } from 'react';
+import { CSSInterpolation } from '@emotion/css';
+import get from 'lodash.get';
+
+import { NotificationCenterStyles, StylesPaths } from './styles-provider.types';
+import { CSSFunctionOrObject } from '../../components/types';
+import { useNovuTheme } from '../../hooks';
+
+const StylesContext = createContext<{ styles: NotificationCenterStyles } | undefined>(undefined);
+
+export const useStyles = (path: StylesPaths | StylesPaths[]): CSSInterpolation[] => {
+  const stylesContext = useContext(StylesContext);
+  const { theme, colorScheme, common } = useNovuTheme();
+
+  if (!stylesContext) {
+    throw new Error('useStyles must be used within a StylesProvider');
+  }
+
+  const getStyleByPath = (pathInStyles: StylesPaths): CSSInterpolation => {
+    const stylePart: CSSFunctionOrObject = get(stylesContext.styles, pathInStyles);
```

Comment:
If we would like to not add the lodash extra dependency we could do:
```
const get = (obj, path) => (
    path.split('.').reduce((acc, level) => acc && acc[level], obj)
);
```

Reply:
wow yeah, good suggestion :) I'll change it